libraries/src/Extension/ExtensionHelper.php

```diff
@@ -263,6 +263,7 @@ class ExtensionHelper
 		array('plugin', 'multilang', 'sampledata', 0),
 
 		// Core plugin extensions - system
+		array('plugin', 'accessibility', system, 0),
```

Comment:
```suggestion
		array('plugin', 'accessibility', 'system', 0),
```

Actually tests are correctly failing.  https://ci.joomla.org:444/artifacts/joomla/joomla-cms/4.0-dev/27940/system-tests/30108/ContentCategoryListCest.Category.mysql.fail.png